tests/integration/utils2.go

```diff
@@ -1355,6 +1359,10 @@ func deleteDoc(
 	}
 
 	assertExpectedErrorRaised(s.t, s.testCase.Description, action.ExpectedError, expectedErrorRaised)
+
+	if action.ExpectedError == "" {
+		waitForUpdateEvents(s, action.NodeID, action.CollectionID)
```

Comment:
question: Why is this only done using create-save, and not create-gql and the other create?  And why not on delete?

thought: This looks quite easy to mess up and forget, and it missing it might not create an obvious and immediate failure (just flaky tests).

Reply:
+1 similar thought, for here and other instances of this change.

Reply:
If I'm understanding this correctly the `createDoc` function calls the `create-save` and `create-gql` functions so they both should be covered.

Reply:
:grin: My bad, the github presentation of the diff helped me miss-read where this line was...  Ignore this thread :)